frontend/src/components/SilvicultureSearch/Openings/SearchScreenDataTable/index.tsx

```diff
@@ -50,8 +52,8 @@ interface ISearchScreenDataTable {
   rows: OpeningsSearch[];
   headers: ITableHeader[];
   defaultColumns: ITableHeader[];
-  handleCheckboxChange: (columnKey: string) => void;  
-  toggleSpatial: () => void;
+  handleCheckboxChange: Function;
```

Comment:
Define the function signature here to avoid issues when consuming a function.

Reply:
done